completions/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+# Copyright (C) 2020  Matthew Glazar
+# See end of file for extended copyright information.
+
+cmake_minimum_required(VERSION 3.10)
+include(GNUInstallDirs)
+
+install(
+  FILES _quick-lint-js
+  DESTINATION "${CMAKE_INSTALL_DATAROOTDIR}/zsh/site-functions"
+)
+
+install(
+  FILES quick-lint-js.bash
+  DESTINATION "${CMAKE_INSTALL_DATAROOTDIR}/bash-completion/completions"
+)
+
+install(
+  FILES quick-lint-js.fish
+  DESTINATION "${CMAKE_INSTALL_DATAROOTDIR}/fish/completions"
```

Comment:
This looks like the wrong installation location for fish completions, according to my reading of this document:

> The completions shipped with fish, usually installed in /usr/share/fish/completions
> https://fishshell.com/docs/current/completions.html

I think fish/vendor_completions.d/ is a better installation location:

> A directory for third-party software vendors to ship their own completions for their software, usually /usr/share/fish/vendor_completions.d

Reply:
fish completions were autogenerated from the `man` page on my machine. This is because my fish installation didn't look in /usr/share/fish/completions; it instead looked in /usr/share/fish/vendor_completions.d.

Reply:
Changed in next commit